src/docs/product/error-monitoring/index.mdx

```diff
@@ -1,7 +1,38 @@
 ---
-title: Error Monitoring
+title: Issues
 sidebar_order: 1
-description: "Learn more about how the configuration of your SDK helps you resolve issues quickly in sentry.io."
+description: "Learn more about the Issues page and how you can use it to efficiently triage issues."
 ---
 
+The Issues page is where you can find information about important groups of errors in your application. Filter by error event properties such as browser, device, impacted users, or whether the error is unhandled. You can then inspect issue details to better understand the problem and [triage](/product/error-monitoring/issue-details/) effectively.
+
+TO DO add screenshot here of issue stream
+
+
+## Error Events and Issues
+
+A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.
+
+From the Issues page, you can begin to triage an issue. Learn more about triaging issues and their different states in [Issue States and Triage](/product/error-monitoring/states-triage/).
+
+For each issue, the page displays:
+
+- Error type and message
+- Brief description of the issue
+- Associated project
+- Issue timing, such as last and first time seen
+
+## Issues Page Tabs
```

Comment:
This section can go away, let's discuss tabs in the triage page under the EA section